src/ResourceManager/ServiceFabric/Commands.ServiceFabric/Commands/UpdateAzureRmServiceFabricDurability.cs

```diff
@@ -125,11 +125,10 @@ public override void ExecuteCmdlet()
                     NodeTypes = cluster.NodeTypes
                 };
 
-                var patchTask = PatchAsync(patchArg);
+                var psCluster = SendPatchRequest(patchArg);
 
-                WriteClusterAndVmssVerboseWhenUpdate(new List<Task>() { vmssTask, patchTask }, true, this.NodeType);
```

Comment:
Why is this change needed? The VMSS and SF updates should be able to run in parallel.